Zoom/Zoom.csproj

```diff
@@ -1,8 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
```

Comment:
<img width="472" alt="image" src="https://github.com/SyncfusionExamples/maui-pdf-viewer-examples/assets/65169077/b1b766c2-f156-4ac9-8f43-41a30a9505b3">

NuGet versions was not changed for some samples. Did you run and check?

Reply:
yes sir run it. it was working